v20.2/performance-benchmarking-with-tpc-c-10k-warehouses.md

```diff
@@ -260,13 +266,13 @@ Next, [partition your database](partitioning.html) to divide all of the TPC-C ta
     postgres://root@<node 1 internal address>:26257?sslmode=disable postgres://root@<node 2 internal address>:26257?sslmode=disable postgres://root@<node 3 internal address>:26257?sslmode=disable postgres://root@<node 4 internal address>:26257?sslmode=disable ...
     ~~~
 
-2. Run TPC-C for 30 minutes:
+2. Run TPC-C for 30 minutes:  (<strong><font style="size:xx-large" color="red">FIXME</font></strong>: is this still the right command?)
 
     {% include copy-clipboard.html %}
     ~~~ shell
     $ ulimit -n 20500 && ./workload run tpcc \
     --partitions 5 \
-    --warehouses 10000 \
+    --warehouses 13000 \
```

Comment:
Same as above - don't know if this is the right command.

(In general, I've done a `s/10,000/13,000/g ` - please yell in places where that did not make sense)